Twilio/Deserialize.php

```diff
@@ -7,16 +7,19 @@ class Deserialize {
     /**
      * Deserialize a string date into a DateTime object
      *
-     * @param string $s A date or date and time, can be iso8601, rfc2822, 
+     * @param string $s A date or date and time, can be iso8601, rfc2822,
      *                  YYYY-MM-DD format.
      * @return \DateTime DateTime corresponding to the input string, in UTC time.
      */
     public static function dateTime($s) {
         try {
-            return new \DateTime($s, new \DateTimeZone('UTC'));
+            if ($s) {
+                return new \DateTime($s, new \DateTimeZone('UTC'));
+            }
         } catch (\Exception $e) {
-            return $s;
+            // no-op
```

Comment:
don't we need to log this as standard practice?

Reply:
We pretty much don't do any logging in the helper libs. I don't like silent-failing _anywhere_, but that's the current status quo.

Reply:
How about capturing this as issues across our repos?

Reply:
Added.